docker/gaffer-kerberos/gaffer-integration-tests-krb/accumulo-int-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Copyright 2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Wait a minute for Accumulo to be started and working
+sleep 60
```

Comment:
Is there not something you can probe in a loop to see if its up rather than hardcoded sleep?

Reply:
It might be possible to reuse what's been done in the healthchecks, I'll have a look.